cli/common.go

```diff
@@ -144,15 +151,27 @@ func AuthGimmeAwsCreds(gacProfile string) error {
 	}
 
 	// execute gimme-aws-creds
-	cmd := exec.Command("gimme-aws-creds", "--profile", gacProfile)
+	cmd := exec.Command("gimme-aws-creds", "--profile", profile.GimmeAWSCreds)
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
 	cmd.Stdin = os.Stdin
 	if err := cmd.Run(); err != nil {
 		return fmt.Errorf("unable to execute `gimme-aws-creds`: %w", err)
 	}
 
-	return nil
+	// Verify we are authenticated to AWS now that we have run gimme-aws-creds
+	if canAuth(profile) {
+		return nil
+	}
+
+	home, err := os.UserHomeDir()
+	if err != nil {
+		home = "~"
```

Comment:
this should be ok on any unix-ey enough OS, but I don't know how portable it is otherwise, versus env var HOME, for example?

Reply:
https://pkg.go.dev/os#UserHomeDir

> On Unix, including macOS, it returns the $HOME environment variable. On Windows, it returns %USERPROFILE%. On Plan 9, it returns the $home environment variable.

It should be fairly portable as-is, although honestly I've never validated if `ax` functions even remotely proper on Windows. The thinking there was if for some reason `os.UserHomeDir()` returns an error just default back to the unix-ey syntax of a `~` to represent the home directory in the error message.

Reply:
eh anyone left on Windows at this point, would be use WSL for this work anyway which would give them bash native syntax